infra/launch.py

```diff
@@ -19,11 +19,18 @@ def setup_vm_docker(tpu_name, zone, docker_base_image):
         "-aG",
         "docker",
         getpass.getuser(),
+        "&&" "docker",
+        "volume",
+        "create",
+        "--driver=local",
+        "levanter",
+        "&&",
```

Comment:
I don’t have an opinion here just noting: previously this would happen regardless and now it skips if previous step fails

Reply:
Yeah, I don't love it, but in theory the behavior should be the same (because `tpu_ssh` by default aborts if the command fails). 

(All of the commands should always succeed. `docker volume create` and `docker rm -f levanter` both always return 0 for most normal situations. It's a no-op if the volume already exists, and `rm -f` returns 0 if the container doesn't exist.)